README.md

```diff
@@ -31,27 +29,29 @@ TODO
 - [Install Brownie](https://eth-brownie.readthedocs.io/en/stable/install.html)
 
 - Install Dev dependencies: `pip3 install -r requirements-dev.txt`
-    - Use whatever pip version avaiable
+
+  - Use whatever pip version avaiable
 
 - Install ganache-cli: `npm install -g ganache-cli@6.12.0`
+
   - > For those of you using Node 14, you'll need to launch ganache-cli via node 8, 10, or 12 until this is fixed. For example, run nvm use 12 && npm install ganache-cli -g to install for node 12, and then run ganache-cli with nvm use 12 && ganache-cli.
 
 - Install npm dependencies: `npm install`
 
 - Install project dependencies:
+
 ```
-$ brownie pm install iearn-finance/yearn-vaults@0.1.1
+$ brownie pm install iearn-finance/yearn-vaults@0.2.0
```

Comment:
Come to think of it, I think Brownie handles this internally (on the first install)

Reply:
yep, you always mention pm and I never used it. I just run `brownie compile`

Reply:
yeah it pulls it for you directly